cdc/entry/codec.go

```diff
@@ -301,3 +309,11 @@ func unflatten(datum types.Datum, ft *types.FieldType, loc *time.Location) (type
 	}
 	return datum, nil
 }
+
+func decodeRowMeta(rowByte []byte, tz *time.Location) (types.Datum, error) {
+	datums, err := tablecodec.DecodeRowToDatumMap(rowByte, rowMetaMap, tz)
+	if err != nil {
+		return types.Datum{}, cerror.WrapError(cerror.ErrDecodeRowToDatum, err)
+	}
+	return datums[metaColID], nil
```

Comment:
`datums[metaColID]` could be nil if the data does not contains meta column

Reply:
fixed, PTAL